src/containers/shared/SocketContext.tsx

```diff
@@ -12,25 +12,37 @@ function isInsecureWs(rippledHost: string | undefined): boolean {
 }
 
 function getSocket(rippledUrl?: string): XrplClient {
```

Comment:
Should `getSocket` accept an optional string array of urls instead of being converted to an array immediately?

Reply:
I kept it this way to make the handling of the url as an environment variable or user specified work the same.